### DISCUSSION TITLE
BigInt fields are not searchable

### DISCUSSION THREAD

**Discussion by davinyoeun** [laravel/nova-issues#6244] [Q&A]
Laravel Version: 10
Nova Version: 4
PHP Version: 8.2

Description:
ID::make('Id')->asBigInt()

When I search record by id it's not show data if id is not match all numbers.

**Comment by crynobone** [comment]
All Primary Key uses exact value search to improve search performance. Using `LIKE` in this case would render SQL skipping index for the ID column.
